libs/wordpress-plugin/plugin/trunk/settings/react-promo.tsx

```diff
@@ -0,0 +1,257 @@
+);
+
+const PromoList = ({ children }: PropsWithChildren) => {
+  return <ul className={styles.PromoList}>{children}</ul>;
+};
+
+const PromoListItem = ({
+  children,
+  comingSoon = null,
+}: PropsWithChildren<{ comingSoon?: ReactNode }>) => {
+  return (
+    <li
+      className={clsx(
+        styles.PromoListItem,
+        comingSoon && styles.PromoListItemComingSoon,
+      )}
+    >
+      {comingSoon ? (
+        <>
+          <FontAwesomeIcon icon={faArrowUpRight} />
+          <div className={styles.PromoListItemComingSoonText}>{comingSoon}</div>
+        </>
+      ) : (
+        <FontAwesomeIcon icon={faArrowRight} />
+      )}
+      {children}
+    </li>
+  );
+};
+
+const PromoInfoText = ({ children }: PropsWithChildren) => (
+  <span className={styles.PromoInfoText}>{children}</span>
+);
+
+const PromoInfoImage = ({
+  src,
+  alt,
+  width,
+}: {
+  src: string;
+  alt?: string;
+  width: number;
+}) => {
+  return (
+    <div style={{ width }} className={styles.PromoInfoImage}>
+      <img src={src} alt={alt} />
+    </div>
+  );
+};
+
+const PromoCardPurpleText = ({ children }: PropsWithChildren) => (
+  <span className={styles.PromoCardPurpleText}>{children}</span>
+);
+
+export const ReactPromo = () => {
+  return (
+    <div className={styles.PromoStack}>
+      <h2 className="BPActivateHeading">You can look forward to...</h2>
+      <PromoCard>
+        <PromoCardHeader>
+          <PromoCardSmallTitle>Next Generation</PromoCardSmallTitle>
+          <PromoCardBigTitle>
+            <PromoCardPurpleText>AI Blocks</PromoCardPurpleText>
+          </PromoCardBigTitle>
+        </PromoCardHeader>
+        <PromoCardItems>
+          <PromoCardContent>
+            <PromoList>
+              <PromoListItem>
+                The best <strong>AI text</strong> generation, style, tone,
+                grammar and editing blocks for WordPress
+              </PromoListItem>
+              <PromoListItem>
+                Powerful <strong>AI image generation</strong> blocks let you
+                illustrate posts in seconds
+              </PromoListItem>
+              <PromoListItem>
+                Advanced <strong>AI image editing</strong> blocks let you remove
+                watermarks from photos, or remove/replace the backgrounds of
+                product photos in WooCommerce
+              </PromoListItem>
+              <PromoListItem>
+                Best-in-class <strong>AI chat</strong> block lets you persist
+                chats, publish them on your website, fork prior conversations
+                with history intact, set the AI’s response style and more
+              </PromoListItem>
+              <PromoListItem>
+                Switch between AI models and providers at will
+              </PromoListItem>
+              <PromoListItem>
+                <strong>
+                  <PromoCardPurpleText>Free credits</PromoCardPurpleText> for
+                  all models and providers
```

Comment:
```suggestion
                  all providers
```

Slight last-minute change on our part means that at least initially we won't be providing free GPT-4 credits due to its slightly more complex billing structure (separate rates for input and output tokens), as well as much higher costs. But "all providers" remains accurate.